static/js/src/advantage/subscribe/reducers/form-reducer.js

```diff
@@ -115,7 +115,9 @@ function getProduct(state) {
 
 const formSlice = createSlice({
   name: "form",
-  initialState: loadState("ua-subscribe-state", "form", initialFormState),
+  initialState: {
+    ...loadState("ua-subscribe-state", "form", initialFormState),
```

Comment:
I'm not sure what `loadState` returns but in this case these two values are pretty much identical if there are nested objects in there.

Reply:
Good catch I was at some point overriding some attributes after the deconstruction but I'm not anymore so I'll revert it